dashboarddoc.md

```diff
@@ -167,14 +167,40 @@ format. Click ‘Choose File’ and select the data file to upload.
 
 Download Data
 -------------
-The dashboard will include tools to allow users to download data from
-an Observation or Forecast page. Users will be able to select the time
-period for which to download data. The format and field names in
-the downloaded data will be the same as for data uploads. 
+{: .anchor}
 
+The instructions here will describe the process of
+downloading data using the dashboard.
 Users may also utilize the API to download data. See the
 [API documentation](https://dev-api.solarforecastarbiter.org/)
 for details.
 
+### Download Observation data
+{: .anchor}
+
+1.  From Sites listing page, click on a site.
+
+2.  Click ‘Observations’ to find the Observations listing for that site.
+
+3.  Select an Observation. Click ‘Download Data’ on the Observation page.
+
+4.  Specify a date range and format, and click download. Start and end
+	times must be in UTC. 
+
+	<img class="my-3" src="/images/obs_download.png">
```

Comment:
why is format a menu here and a radio button with examples above? I prefer the radio button with examples (default csv). This would also make it clear that the downloaded time is in UTC.

Reply:
Or we could be helpful and add the timezone menu like on the site creation page. Default could be UTC.